contrib/sly-package-fu.el

```diff
@@ -317,4 +332,126 @@ symbol in the Lisp image if possible."
 
 (defalias 'sly-export-structure 'sly-export-class)
 
+;; 
+;; Dealing with import-from
+;;
+;; MELPA version: (buffer-substring-no-properties point1 pount2)
+;; ~/.emacs.d/elpa/sly-20171111.1604/contrib/sly-package-fu.el
```

Comment:
I can't understand these two lines of comments

Reply:
Sorry, I've removed this junk :)